var/spack/repos/builtin/packages/fastor/package.py

```diff
@@ -0,0 +1,28 @@
+# Copyright 2013-2024 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+
+from spack.package import *
+
+
+class Fastor(CMakePackage):
+    """Fastor is a lightweight high performance tensor algebra framework
+    for modern C++."""
+
+    homepage = "https://github.com/romeric/Fastor"
+    url = "https://github.com/romeric/Fastor/archive/refs/tags/V0.6.4.tar.gz"
+
+    maintainers("wdconinc")
+
+    license("MIT", checked_by="wdconinc")
+
+    version("0.6.4", sha256="c97a3b9dbb92413be90689af9d942cddee12a74733cf42f1a8014965553a11f8")
+
+    depends_on("cmake@3.20:", type="build")
+
+    def cmake_args(self):
+        args = [
+            self.define("BUILD_TESTING", self.run_tests),
+        ]
+        return args
```

Comment:
```suggestion
        return [
            self.define("BUILD_TESTING", self.run_tests),
        ]
```

Optional suggestion to drop the variable declaration.

Reply:
Thanks! Fixing style while I'm at it with:
```suggestion
        return [self.define("BUILD_TESTING", self.run_tests)]
```

Reply:
Aside: maybe `BUILD_TESTING` should be set in `lib/spack/spack/build_systems/cmake.py` for consistency?